charts/kyverno/templates/_helpers/_securityContext.tpl

```diff
@@ -0,0 +1,9 @@
+{{/* vim: set filetype=mustache: */}}
+
+{{- define "kyverno.securityContext" -}}
+{{- if semverCompare "<1.19" .version -}}
```

Comment:
Does Kyverno still support `< 1.19`?

Reply:
Nope, shall I drop it ?

Reply:
I think so.  I think the only situation where it would be good to keep is if things like OpenShift or Rancher use older Kubernetes versions that they still support.  It appears for at least OpenShift the supported versions use newer Kubernetes so should be fine to remove this condition.

Reply:
Fixed